### PR TITLE
readme typos

### PR DESCRIPTION
**Pull request by JKhawaja**
**- What I did**

Corrected English grammar mistakes in README

**- How I did it**

I forked the repo and edited the README file right here on GitHub

**- How to verify it**

Find an English language scholar from a top academic University and verify that my English grammar is accurate and professional.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**- Description for the changelog**

### CODE REVIEW

**Comment by JKhawaja** [comment]
I just discovered Bloom and it looks very ambitious but very awesome!

File: README.md [hunk 1]
```diff
@@ -8,11 +8,11 @@
 </p>
 
 
-You no longer trust tech monopolies with your data?<br/>
-You are done with your privacy invaded by advertisers? <br/>
-You are tired of being abused by Big Companies?
+Do you no longer trust tech monopolies with your data?<br/>
+Are you tired of your privacy being invaded by advertisers? <br/>
```
Comment (skerkour) [inline README.md:12]: Can you please replace tired by done.

Comment (JKhawaja) [reply README.md]: Done. (pun intended)

**Comment by skerkour** [comment]
Thank you very much!

**Pull request merged**